android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -540,7 +554,6 @@ public void receiveCommand(WebView root, int commandId, @Nullable ReadableArray
         if (args == null) {
           throw new RuntimeException("Arguments for loading an url are null!");
         }
-        root.loadUrl(args.getString(0));
```

Comment:
How does LOAD_URL command works now?

Reply:
This is my bad - accidently removed while I was pulling from upstream.